nixos/modules/services/mail/postfix.nix

```diff
@@ -719,44 +766,33 @@ in
             ExecStart = "${pkgs.postfix}/bin/postfix start";
             ExecStop = "${pkgs.postfix}/bin/postfix stop";
             ExecReload = "${pkgs.postfix}/bin/postfix reload";
+            ExecStartPre = "+${preStartScript}";
+
+            ReadWritePaths = [ "/var/lib/postfix" "/var/spool/mail" ];
+
+            CapabilityBoundingSet = [ "CAP_DAC_OVERRIDE" "CAP_NET_BIND_SERVICE" "CAP_SETGID" "CAP_SETUID" ];
+            DevicePolicy = "closed";
+            LockPersonality = true;
+            MemoryDenyWriteExecute = true;
+            NoNewPrivileges = true;
+            PrivateDevices = true;
+            PrivateMounts = true;
+            PrivateTmp = true;
+            ProtectClock = true;
+            ProtectControlGroups = true;
+            ProtectHostname = true;
+            ProtectKernelLogs = true;
+            ProtectKernelModules = true;
+            ProtectKernelTunables = true;
+            ProtectSystem = "full";
```

Comment:
Does this still allow other milters? I know that our rspamd integration for postfix for example creates a socket in `/run/rspamd/rspamd-milter.sock`, opendkim might be similar.

Reply:
@GrahamcOfBorg test rspamd

Reply:
IIUC, the `full` value still allows RW accesses to `/run`. 
Also, I've run the nixos-mailserver tests (rspamd, opendkim, dovecot,...) on top of this patch and tests passed.